config/workflow_config.json

```diff
@@ -17,49 +17,19 @@
   },
   {
     "workflow_type_name": "simulator",
-    "workflow_type_description_name": "High fidelity simulator"
-  },
-  {
-    "workflow_type_name": "test_worker",
-    "workflow_type_description_name": "Used for testing purposes. Should not be used in production environments.",
+    "workflow_type_description_name": "High fidelity simulator",
     "workflow_parameters": [
       {
-        "parameter_type": "datetime",
-        "key_name": "start_time"
+        "parameter_type": "duration",

```

Comment:
I would expect `number_of_steps` next to `start_time` and `end_time`.  
Also, since `duration` will be an integer (or number?) input field it would be nice to provide more information, like a `description` to specify the units and possibly a `minimum` and `maximum` value.

Reply:
Description added through https://github.com/Project-OMOTES/orchestrator/pull/66/commits/29e546e44eda055ae22b8a05d88a1b09f7728184

Created a ticket to add a minimum and maximum for the DurationParameter in the SDK here: https://github.com/Project-OMOTES/omotes-sdk-python/issues/71  and here https://github.com/Project-OMOTES/omotes-sdk-protocol/issues/18

Reply:
> I would expect `number_of_steps` next to `start_time` and `end_time`.

The timestep duration together with start_time and end_time already define a profile including the number of the steps in the profile right? Currently the simulator worker doesn't require any other parameters: https://github.com/Project-OMOTES/simulator-worker/blob/main/src/simulator_worker/simulator_worker.py#L85

Reply:
Added minimum field here: https://github.com/Project-OMOTES/orchestrator/pull/66/commits/2cd3b19855bb7f9009d527fe83f97de05ade56ac

Reply:
It is clear now, thanks